pkg/api/handlers/compat/containers.go

```diff
@@ -95,9 +95,10 @@ func ListContainers(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value("runtime").(*libpod.Runtime)
 	decoder := r.Context().Value("decoder").(*schema.Decoder)
 	query := struct {
-		All   bool `schema:"all"`
-		Limit int  `schema:"limit"`
-		Size  bool `schema:"size"`
+		All     bool   `schema:"all"`
+		Limit   int    `schema:"limit"`
+		Size    bool   `schema:"size"`
+		Filters string `schema:"filters"`
```

Comment:
Filters should already work. They are handled differently, check below for `util.PrepareFilters(r)`

Reply:
good point, I will look into why it isn't picking up on status then. was thinking it was a simple missing query param, looks like it might be deeper or just an issue a few lines later in the compat code.